gradle/dependency-versions.gradle

```diff
@@ -34,7 +34,7 @@
   jodaTimeVersion = "2.2"
   joptSimpleVersion = "3.2"
   junitVersion = "4.12"
-  kafkaVersion = "0.11.0.2"
+  kafkaVersion = "2.0.1"
```

Comment:
Discussed offline.

Since we're bumping up kafka-client version to 2.0, it would be better to run the  perf tests. If possible try running [kafka-read-write-perf](https://github.com/apache/samza/blob/master/samza-test/src/main/config/perf/kafka-read-write-performance.properties) test before landing this patch and verify there's no performance degradation(in terms of QPS/poll-latency).

Reply:
will do